api/v1alpha1/common.go

```diff
@@ -543,3 +543,16 @@ type LogConfig struct {
 	Name string `json:"name"`
 	Key  string `json:"key"`
 }
+
+type WindowConfig struct {
+	// +kubebuilder:validation:Required
+	ActualWindowFunctionClassName string  `json:"actualWindowFunctionClassName"`
```

Comment:
Is it possible to let the user not set the `ActualWindowFunctionClassName`, and uses the `className` instead when constructing the container command? Otherwise, the user needs to set `className` to the window function executor and set `ActualWindowFunctionClassName` to the function class name, which is not very friendly.

Reply:
got it